fiftyone/utils/annotations.py

```diff
@@ -2279,7 +2279,7 @@ def _prompt_api_key(self, backend):
             prefix = "FIFTYONE_%s_" % backend.upper()
             logger.info(
                 "Please enter your API key.\nYou can avoid this in the future "
-                "by setting your `%sKEY` environment variable",
+                "by setting your `%sAPI_KEY` environment variable",
```

Comment:
@ehofesmann this is a universal bug, not just labelbox, eh? Proposing to pull it into `release/v0.23.8` immediately just in case here: https://github.com/voxel51/fiftyone/pull/4256

Reply:
Believe universal yes for annotation integrations

Reply:
ah you added this @allenleetc, thanks 💪

Reply:
Yes, universal for all API key-based backends it seems (everything but CVAT). Good catch Allen!